package.json

```diff
@@ -29,10 +29,13 @@
     "@oceanprotocol/lib": "^2.6.1",
     "@oceanprotocol/typographies": "^0.1.0",
     "@oceanprotocol/use-dark-mode": "^2.4.3",
+    "@sphereon/ssi-sdk-qr-react": "^0.9.0",
```

Comment:
Do not mix ssi-sdk versions. This should be ssi-sdk. like the rest with version 0.11.1-next.70
Same for the next line. Never mix versions of the SDK

Lastly. Why do you need the kv-store for the REST client and QR code generator?

Reply:
Fixed!